### ISSUE TITLE
Request : Font attributes

### ISSUE THREAD

**Issue by rodnem** [canisminor1990/sketch-select#15]
Hello, 
Sometimes you need to reassign text styles… it's a pain!

Could Sketch-Select perform more precise selection by attributes in Text tab?
Cumulative selection with same :
• Typeface
• Weight
• font color
• font-size
• Alignment
• Spacing

Thanks in advance

**Comment by AdamSzakal** [comment]
+1
This was a feature in the preivous versions – not sure why it was removed?

**Comment by ArttieZhou** [comment]
+1 !

**Comment by tacosauce** [comment]
Yes this was the main reason I used this plugin to just mass edit the color of text. It keeps saying "fill color needs to set" because of the new sketch updates. "Fill" is not the same as text color

**Comment by luisjmontoro** [comment]
+1! It was very useful for cleaning unwanted text styles on a document.

**Comment by agaalberto** [comment]
> +1! It was very useful for cleaning unwanted text styles on a document.

+1
